src/trading/IOrders.sol

```diff
@@ -5,31 +5,31 @@ import 'ROOT/reporting/IMarket.sol';
 
 
 contract IOrders {
-    function saveOrder(Order.TradeTypes _type, IMarket _market, uint256 _fxpAmount, int256 _fxpPrice, address _sender, uint8 _outcome, uint256 _fxpMoneyEscrowed, uint256 _fxpSharesEscrowed, bytes32 _betterOrderId, bytes32 _worseOrderId, uint256 _tradeGroupId) public returns (bytes32 _orderId);
+    function saveOrder(Order.TradeTypes _type, IMarket _market, uint256 _fxpAmount, uint256 _price, address _sender, uint8 _outcome, uint256 _moneyEscrowed, uint256 _sharesEscrowed, bytes32 _betterOrderId, bytes32 _worseOrderId, uint256 _tradeGroupId) public returns (bytes32 _orderId);
```

Comment:
I wonder if we should come up with a better name for the `price` variable?  In the reporting system we called the denominator `payoutDenominator` and the numerator `payoutNumerator` and the two combined to be a payout multiplier (as a rational number).  Similarly here, the denominator (`marketDenominator`) combined with `<TBD>` results in a price multiplier (as a rational number).  I'm worried that someone looking at the name `price` won't have much of a chance of guessing what they need to put here.  What they need is a number between `0` and `marketDenominatory` that represents the distribution of share value between maker and taker.

Reply:
`marketPrice` maybe?